starcounter-upload.html

```diff
@@ -66,6 +66,10 @@
                     type: Number,
                     value: 8192
                 },
+                maxFileSize: {
```

Comment:
Please add some comments, describing in which unit the value should be specified, and how to disabled it: use `-1` or `0`.

P.S. I think, the common practice is to use `0` in this case.